10-workload.md

```diff
@@ -49,7 +49,7 @@ The cluster now has an [Traefik configured with a TLS certificate](./08-secret-m
    exit
    ```
 
-   > :beetle: If you are running a version of kubectl less than 1.23, you'll receive an error from the run command above as the method to provide container limits has changed between 1.22 and 1.23. On kubectl versions less than 1.23, you'll need to run the following command instead and you can [safely ignore the message about `--limits` being deprecated](https://github.com/kubernetes/kubectl/issues/1101).
+   > :beetle: If you are running a version of kubectl less than 1.23, you'll receive an error from the run command below as the method to provide container limits has changed between 1.22 and 1.23. On kubectl versions less than 1.23, you'll need to run the following command instead and you can [safely ignore the message about `--limits` being deprecated](https://github.com/kubernetes/kubectl/issues/1101).
```

Comment:
Eh?  But that's not what we're trying to say.  (adding this just as a tracker for myself)

Reply:
reverted this based on our offline convo 

done | addressed from 3a9c88e6c60eac9033298568662653c97686128c